wf_environment/wf_environment.module

```diff
@@ -891,3 +903,15 @@ function wf_environment_delete_form_submit($form, &$form_state) {
   drupal_set_message(t("Environment '@title' was successfully deleted.", array('@title' => $title)));
   $form_state['redirect'] = url('admin/content/environments');
 }
+
+/**
+ * Gets related job count from a environment.
+ */
+function wf_environment_get_job_count($env_id) {
+  $results = db_select('wf_job', 'wf_job')
+    ->fields('wf_job')
+    ->condition('eid', $env_id)
+    ->execute()
+    ->rowCount();
```

Comment:
This is inefficient as it iterates over the result set. Please use [countQuery](https://www.drupal.org/docs/7/api/database-api/dynamic-queries/count-queries) like so:

```php

$results = db_select('wf_job', 'wf_job')
    ->fields('wf_job')
    ->condition('eid', $env_id)
    ->countQuery()
    ->execute()
    ->fetchField();
```

This will use the database's native `COUNT()` function.